### PR TITLE
CNV-40088: fix dashboard alert popup

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

- loading and empty state were showing together for some seconds (when loaded is false)
- do not show Critical and warning if there are 0 critical or 0 warning alerts

## 🎥 Demo

**Before**

Screenshot 2024-07-09 at 10 42 06

**After**

Screenshot 2024-07-09 at 10 37 45

### CODE REVIEW

File: src/views/dashboard-extensions/KubevirtHealthPopup/KubevirtHealthPopup.tsx [hunk 1]
```diff
@@ -30,32 +32,35 @@ const KubevirtHealthPopup: FC = () => {
...
-            </div>
-          </StackItem>
-          <StackItem>
-            <div className="kv-health-popup__alerts-count">
-              <YellowExclamationTriangleIcon className="kv-health-popup__alerts-count--icon" />{' '}
-              <Link to={`${ALERTS_BASE_PATH}${HealthImpactLevel.warning}`}>
-                {`${alerts?.[AlertType.warning]?.length} Warning`}
-              </Link>
-            </div>
+            <div className="kv-health-popup__title">Alerts</div>
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply src/views/dashboard-extensions/KubevirtHealthPopup/KubevirtHealthPopup.tsx]: fixed everything. Thanks @avivtur

File: src/views/dashboard-extensions/KubevirtHealthPopup/components/HealthPopupChart.tsx [hunk 2]
```diff
@@ -6,11 +6,14 @@ import { ChartDonut } from '@patternfly/react-charts';
...
 
+import EmptyStateNoAlerts from './EmptyStateNoAlerts';
+
 type HealthPopupChartProps = {
   alerts: AlertsByHealthImpact;
+  numberOfAlerts?: number;
 };
 
-const HealthPopupChart: React.FC<HealthPopupChartProps> = ({ alerts }) => {
+const HealthPopupChart: React.FC<HealthPopupChartProps> = ({ alerts, numberOfAlerts }) => {
```
Comment (avivtur) [inline src/views/dashboard-extensions/KubevirtHealthPopup/components/HealthPopupChart.tsx:16]: Please remove React.

**Review by avivtur — APPROVED** [review]

**Review by avivtur — APPROVED** [review]

**Pull request merged**